### PR TITLE
[prometheus-druid-exporter] synced with upstream helm chart, updated the image tag and fixed a typo

### PR DESCRIPTION
**Pull request by applike-ss**
#### What this PR does / why we need it:
Sync with the upstream repo the original helm chart came from.
Additionally i updated the image used to reflect the chart version (0.11 => v0.11).
This adds the possibility to disable the `druid_emitted_metrics_histogram_*` metrics which eat up quite a lot of space/memory on our prometheus.
Also fixed a typo from the upstream chart in the connection test.

#### Which issue this PR fixes
This adds the possibility to disable the `druid_emitted_metrics_histogram_*` metrics which eat up quite a lot of space/memory on our prometheus.

#### Special notes for your reviewer:
None

#### Checklist
- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-druid-exporter/Chart.yaml
Comment (monotek) [inline charts/prometheus-druid-exporter/Chart.yaml:7]: new feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-druid-exporter/Chart.yaml
Comment (applike-ss) [reply charts/prometheus-druid-exporter/Chart.yaml:7]: i did already, recheck the committed code please

Comment (monotek) [reply charts/prometheus-druid-exporter/Chart.yaml:7]: No, you changed the patch version. See: https://semver.org/

```suggestion
version: 0.12.0
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Comment by applike-ss** [comment]
sorry, my mistake. Corrected that and pushed again.

File: charts/prometheus-druid-exporter/values.yaml
Comment (monotek) [inline charts/prometheus-druid-exporter/values.yaml:11]: why is this deleted?

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-druid-exporter/values.yaml
Comment (applike-ss) [reply charts/prometheus-druid-exporter/values.yaml]: Did re-add those and added resources to the deployment that were mentioned in the values.yaml but not used.

**Review by monotek — CHANGES_REQUESTED** [review]
rebase with main still missing.

**Comment by applike-ss** [comment]
> rebase with main still missing.

i did rebase with upstreams main, but i also adjusted the maintainers in the chart.yaml. Should i revert those?

**Comment by monotek** [comment]
I guess one of the current chart maintainers @iamabhishek-dubey / @sandy724 should review to decide if new maintainer should be added.

**Comment by applike-ss** [comment]
It seems the owners are unavailable? Or on holiday?

**Review by monotek — APPROVED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
Please fix test.

**Comment by applike-ss** [comment]
i didn't change the connection test, so not sure why it doesn't work. Do you know why it is not working?

**Comment by monotek** [comment]
But you seemed to have enabled this?
https://github.com/prometheus-community/helm-charts/blob/55528ef44ac19e6391f59d55ab5b2f0e4ee27be7/charts/prometheus-druid-exporter/values.yaml#L67

So before the test never run.
Maybe we jsut need to raise the timeout a bit or add some retries.

**Comment by applike-ss** [comment]
> But you seemed to have enabled this?
> 
> https://github.com/prometheus-community/helm-charts/blob/55528ef44ac19e6391f59d55ab5b2f0e4ee27be7/charts/prometheus-druid-exporter/values.yaml#L67
> 
> So before the test never run. Maybe we jsut need to raise the timeout a bit or add some retries.

i did also add the option to the k8s resource, so beforehand it was always run as well.
Can you re-run the test? I think, i might not have the permission to do so.

**Comment by applike-ss** [comment]
@monotek can you please re-run the test to ensure it's not just flaky?

**Comment by monotek** [comment]
did not help.

**Comment by applike-ss** [comment]
sorry to have wasted your time, but i don't really see value in trying to fix this when druid (hopefully soon) in the next release would contain the metrics we'd like to have that are currently only in this exporter. Closing this PR for that reason. If anyone likes to see this integrated, feel free to copy the code and fix whatever made the connection test fail.

**Pull request closed without merging**
